### DISCUSSION TITLE
Stutters on the latest osu! build

### DISCUSSION THREAD

**Discussion by krsmll** [ppy/osu#26953] [Q&A]
I am having an issue with dropping to very low fps for split seconds. I have tried builds [2023.1231.0](https://github.com/ppy/osu/releases/tag/2023.1231.0) and [2023.1130.0](https://github.com/ppy/osu/releases/tag/2023.1130.0) and I do not get any noticeable stutters.

![fpsdrop](https://github.com/ppy/osu/assets/57319032/f3cbe730-6772-42c8-a3ba-045d868e1530)

Logs are attached, please do let me know if you require anything else.

[logs.zip](https://github.com/ppy/osu/files/14145790/logs.zip)

Video of a massive freeze when enabling hidden for example:

https://github.com/ppy/osu/assets/57319032/57914b15-4cca-4345-8dd5-0fbad998802b

**Comment by peppy** [comment]
Does turning off "high precision input mouse" with this?

![2024-02-19 11 12 26@2x](https://github.com/ppy/osu/assets/191335/d942d4b3-1c36-4cb6-b39d-03ab0f2f5e9e)

**Reply by krsmll** [reply]
Nope, already off.
